src/SslCertificate.php

```diff
@@ -62,6 +66,7 @@ public static function der2pem($der_data): string
 
     public function __construct(
         protected array $rawCertificateFields,
+        private array $publicKeyDetail = [],
```

Comment:
By positioning this an argument between existing ones, this is a breaking change. Could you move this to the last position, so this PR becomes non breaking?